matminer/featurizers/site.py

```diff
@@ -804,15 +801,19 @@ def fit(self, structs):
         in a dict of self.el_amt_dict_, avoiding repeated calculation of
         composition when featurizing multiple sites in the same structure.
         Args:
-            structs (pandas series): series of pymatgen Structures.
+            X (pandas series): series of pymatgen Structures.
+            y : to be compatible with the overridden fit method.
```

Comment:
for docstring of y, put:

y : unused (added for consistency with overridden method signature)